scalastyle-config.xml

```diff
@@ -240,6 +240,18 @@ This file is divided into 3 sections:
     ]]></customMessage>
   </check>
 
+  <check customId="notthrowoutofmemory" level="error" class="org.scalastyle.file.RegexChecker" enabled="true">
```

Comment:
not a big deal but I would name it `nothrowoutofmemory`

Reply:
Thanks, new rule named `throwerror`.